### ISSUE TITLE
Slate does not work with screen readers

### ISSUE THREAD

**Issue by turt2live** [ianstormtaylor/slate#2793]
#### What's the current behavior?

I'm using JAWS on Windows 10 to verify our app works with screen readers, and found that Slate does not work well at all. A minimal example can be found on the live demo page.

The screen reader appears to be losing focus after pressing the Enter key - while Slate does correctly insert a newline character, the screen reader starts reading back the page and further keystrokes result in hotkeys for the screen reader. Using a regular text area (like the one for submitting this issue) does not have the same problem.

#### What's the expected behavior?

The input should not lose the screen reader's focus after pressing Enter.
